app/controllers/[scope]/confirmations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class [scope]::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
@kalgibhatt-btc How come this folder is named `[scope]`, is it some script mistake? I see that all following controllers are under this module.